src/cli/mod.rs

```diff
@@ -88,6 +92,7 @@ pub async fn init() -> ExitCode {
 		Commands::Version(args) => version::init(args).await,
 		Commands::Upgrade(args) => upgrade::init(args).await,
 		Commands::Sql(args) => sql::init(args).await,
+		Commands::Ml(args) => ml::init(args).await,
```

Comment:
Wouldn't it be better, API wise, to add an `--ml` flag to the already existing `import` and `export` commands? This would also be more consistent with the planned Rust API:

```rust
// Export SurrealML models
db.export("path/to/file").ml(name, version).await?;

// Import SurrealML models
db.import("path/to/file").ml().await?;
```

Reply:
The CLI API would then become something like this:

```bash
# Export SurrealML models
surreal export --ml --namespace namespace --database database path/to/file

# Import SurrealML models
surreal import --ml --name name --version version --namespace namespace --database database path/to/file
```

Reply:
There will be additional `ml` commands in the future @rushmorem, so it's better to separate them.